crates/red_knot_python_semantic/src/semantic_index/use_def/bitset.rs

```diff
@@ -119,6 +102,19 @@ impl<const B: usize> BitSet<B> {
             current_block: blocks[0],
         }
     }
+
+    pub(super) fn iter_rev(&self) -> ReverseBitSetIterator<'_, B> {
+        let num_blocks = self.blocks().len();
+
+        debug_assert!(num_blocks > 0);
```

Comment:
I'm leaning towards making this a hard assert and add a debug message. It shouldn't add any significant performance cost
 because it just replaces the bounds check on line 115 with the assert here.